src/server/project.ts

```diff
@@ -1041,6 +1041,14 @@ namespace ts.server {
             super(externalProjectName, ProjectKind.External, projectService, documentRegistry, /*hasExplicitListOfFiles*/ true, languageServiceEnabled, compilerOptions, compileOnSaveEnabled);
         }
 
+        addRoot(info: ScriptInfo) {
+            // Exclude project-included .min.js files
+            if (info.fileName.match(/\.min\.js$/i) !== null) {

```

Comment:
As we talked offline, we probably should mark the file similar to `hasMixedContent`, to ensure that ATA still works, but we are not reading the file.